pkgs/applications/version-management/git-and-tools/radicle-upstream/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, appimageTools, gsettings-desktop-schemas, gtk3, patchelf, zlib, glibc, fetchurl }:
+
+let
+  pname = "radicle-upstream";
+  version = "0.1.4";
+  name = "${pname}-${version}";
+in
+
+appimageTools.wrapAppImage {
+  inherit name;
+  extraPkgs = pkgs : [];
```

Comment:
I am not that familiar with appimageTools but did you try dropping this? IT should default to an empty array if it is undefined.